R/content.R

```diff
@@ -925,3 +975,57 @@ get_content_permissions <- function(content, add_owner = TRUE) {
   res <- content$permissions(add_owner = add_owner)
   parse_connectapi_typed(res, connectapi_ptypes$permissions)
 }
+
+#' Render a content item.
+#' 
+#' @description Submit a request to render a content item. Once submitted, the
+#' server runs an asynchronous process to render the content. This might be
+#' useful if content needs to be updated after its source data has changed,
+#' especially if this doesn't happen on a regular schedule.
+#' 
+#' Only valid for rendered content (e.g., most Quarto documents, Jupyter
+#' notebooks, R Markdown reports).
+#' 
+#' @param content The content item you wish to render.
+#' @return A [VariantTask] object that can be used to track completion of the render.
+#' @export
```

Comment:
Could you add a (not-run) example that shows how you'd use this, especially with the VariantTask?

Reply:
Will do.